maven/templates/rules.bzl

```diff
@@ -337,7 +340,7 @@ assemble_maven = rule(
             doc = 'Project URL to fill into pom.xml'
         ),
         "license": attr.string(
-            values=["apache"],
+            values=["apache", "MIT"],
```

Comment:
@natansil could we use `mit` here (lowercase)? we're trying to achieve consistent naming everywhere